metaflow/runner/click_api.py

```diff
@@ -136,31 +136,76 @@ def get_inspect_param_obj(p: Union[click.Argument, click.Option], kind: str):
     )
 
 
-# Cache to store already loaded modules
-loaded_modules = {}
+# Cache to store already parsed modules
+parsed_modules = {}
+
+
+class FlowSpecVisitor(ast.NodeVisitor):
+    def __init__(self):
+        self.parameters = []
+
+    def construct_parameter(self, node):
+        all_values = {}
+        fields = ["name", "default", "type", "help", "required", "show_default"]
```

Comment:
we use other parameter values so would need something a bit more generic I believe.